src/smolvm/cli/main.py

```
@@ -1161,6 +1081,175 @@ def _run_list(*, include_all: bool, status_filter: str | None, json_output: bool
+
+def _query_live_vm_info(vm_id: str) -> dict[str, object]:
+    """Query a running VM via SSH for OS pretty-name and used memory.
+
+    Returns an empty dict on any failure so the caller can render fall-through
+    placeholders without surfacing transient SSH errors to the user.
+    """
+    from smolvm.facade import SmolVM
+
+    try:
+        vm = SmolVM.from_id(vm_id)
+    except Exception:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)